### PR TITLE
Add some more coroutine builders for `ServiceCall`

### PR DESCRIPTION
**Pull request by gnuzzz**

### CODE REVIEW

**Comment by gnuzzz** [comment]
@ihostage need review

File: buildSrc/src/main/kotlin/Variables.kt [hunk 1]
```diff
@@ -3,6 +3,7 @@ import org.gradle.api.Project
 val Project.isRelease get() = this.version.toString().endsWith("SNAPSHOT")
 
 val Project.lagomVersion get() = this.properties["lagomVersion"] as String? ?: Versions.lagom
+val Project.lagomPac4jVersion get() = this.properties["lagomPac4jVersion"] as String? ?: Versions.lagomPac4j
```
Comment (ihostage) [inline buildSrc/src/main/kotlin/Variables.kt:6]: This changes non needed

File: java/src/test/kotlin/org/taymyr/lagom/javadsl/api/ServiceCallBuildersTest.kt [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+import org.pac4j.core.context.WebContext
+import org.pac4j.core.credentials.UsernamePasswordCredentials
+import org.pac4j.core.exception.HttpAction
+import org.pac4j.core.profile.CommonProfile
+import org.pac4j.core.redirect.RedirectAction
+import org.pac4j.lagom.javadsl.SecuredService
+
+class ServiceCallBuildersTest {
+
+    @Test
```
Comment (ihostage) [inline java/src/test/kotlin/org/taymyr/lagom/javadsl/api/ServiceCallBuildersTest.kt:23]: Please, to add @DisplayName for all tests.

File: java/src/test/kotlin/org/taymyr/lagom/javadsl/api/ServiceCallBuildersTest.kt [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+                val expectedHeader = headers.getHeader(expectedHeaderName)
+                assertThat(expectedHeader.isPresent)
+                assertThat(expectedHeader.get()).isEqualTo(expectedHeaderValue)
+                Pair.create(ResponseHeader.OK, expectedResult)
+            }
+        }
+        runBlocking {
+            val actualResult = simpleService.testMethod()
+                    .handleRequestHeader { headers -> headers.withHeader(expectedHeaderName, expectedHeaderValue) }
+                    .invoke().await()
```
Comment (ihostage) [inline java/src/test/kotlin/org/taymyr/lagom/javadsl/api/ServiceCallBuildersTest.kt:119]: Maybe create extension function `await() = invoke().await()` for `ServiceCall`?

**Review by ihostage — CHANGES_REQUESTED** [review]

**Review by ihostage — APPROVED** [review]
LGTM

**Pull request merged**